zenoh-ts/src/encoding.ts

```diff
@@ -0,0 +1,380 @@
+//
+// Copyright (c) 2024 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+enum encoding {
+  ZENOH_BYTES = "zenoh/bytes",
+  ZENOH_INT8 = "zenoh/int8",
```

Comment:
All of `ZENOH_{U}INT*`, `ZENOH_FLOAT*`, `ZENOH_BOOL`, `ZENOH_ERROR` will be removed in the next release: https://github.com/eclipse-zenoh/zenoh/pull/1512